specs/policy/attributes/attributes.openapi.yaml

```diff
@@ -152,9 +154,11 @@ paths:
             title: with_resource_maps
         - name: withValue.withAttribute.withKeyAccessGrants
           in: query
+          description: Deprecated
           schema:
             type: boolean
             title: with_key_access_grants
+            description: Deprecated
```

Comment:
Do you want to include the recommended approach for this as well (like you did other places)?

Reply:
See https://github.com/opentdf/docs/pull/143#discussion_r2298665001